src/CloudCommunications/beta/examples/Get-MgBetaUserOnlineMeeting.md

```diff
@@ -1,46 +1,34 @@
-### Example 1: Retrieve an online meeting by meeting ID
+### Example 1: Code snippet
 
 ```powershell
 Import-Module Microsoft.Graph.Beta.CloudCommunications
 
 # A UPN can also be used as -UserId.
-Get-MgBetaUserOnlineMeeting -UserId $userId -OnlineMeetingId $onlineMeetingId
-```
-This example shows how to use the Get-MgBetaUserOnlineMeeting Cmdlet.
-
-To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
-
-### Example 2: Retrieve an online meeting by joinWebUrl
-
-```powershell
-Import-Module Microsoft.Graph.Beta.CloudCommunications
-
-# A UPN can also be used as -UserId.
-Get-MgBetaUserOnlineMeeting -UserId $userId -Filter "JoinWebUrl eq 'https://teams.microsoft.com/l/meetup-join/19:meeting_MGQ4MDQyNTEtNTQ2NS00YjQxLTlkM2EtZWVkODYxODYzMmY2@thread.v2/0?context"
+Get-MgBetaUserOnlineMeeting -UserId $userId -Filter "joinMeetingIdSettings/joinMeetingId eq '1234567890'"
 ```
 This example shows how to use the Get-MgBetaUserOnlineMeeting Cmdlet.
 
 To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
 
-### Example 3: Retrieve an online meeting by joinMeetingId
+### Example 2: Code snippet
 
 ```powershell
 Import-Module Microsoft.Graph.Beta.CloudCommunications
 
 # A UPN can also be used as -UserId.
-Get-MgBetaUserOnlineMeeting -UserId $userId -Filter "joinMeetingIdSettings/joinMeetingId eq '1234567890'"
+Get-MgBetaUserOnlineMeetingAttendeeReport -UserId $userId -OnlineMeetingId $onlineMeetingId
```

Comment:
The example shown does not match the file name. `Get-MgBetaUserOnlineMeeting.md` vs `Get-MgBetaUserOnlineMeetingAttendeeReport`.

Reply:
Thanks for flagging this. I've noticed that the script was checking against the api reference for examples with patterns that match the command. Ideally even if there is a pattern that matches the command, it ought to have checked for an exact match. i.e. Matching a string (Command) within boundaries. 
It also needs to retain correct examples if the script contained a mixture of wrong and correct examples.
The latest commit I've done, addresses the above issues.